wfsim/strax_interface.py

```diff
@@ -592,10 +592,9 @@ def get_instructions(self):
         if 'nveto' in self.config['targets']:
             self.instructions_nveto, self.nveto_channels, self.nveto_timings =\
                 read_optical(self.config_nveto)
-            # Why epix removes many of the g4ids?
-            # Remove nveto event if no tpc event of the same g4id is found
             if len(self.g4id) > 0:
                 nv_inst_to_keep = (self.instructions_nveto['_last'] - self.instructions_nveto['_first']) >= 0
+
             self.instructions_nveto = self.instructions_nveto[nv_inst_to_keep]
             self.g4id.append(self.instructions_nveto['g4id'])
             log.debug("Epix produced %d instructions in nv" % (len(self.instructions_nveto)))
```

Comment:
"Epix" is still not true here :)

Reply:
Sorry, I cannot understand your comment. Which epix in the code?
Do you mean we need to  keep `# Why epix removes many of the g4ids?` this comment?

Reply:
It was a minor comment. "Epix produced %d instructions in nv" is not correct, because nVeto entries and epix don't talk to each other.

Reply:
OK, I see.